src/decorators/Observable.jsx

```diff
@@ -17,8 +17,18 @@ const binderRecordEvent = Binder.recordEvent;
 const binderRecordChange = Binder.recordChange;
 const PropertyDefaultValue = { defaultValue: true };
 
+const symbolMap = {};
+function getSymbol(property) {
```

Comment:
Could this be replaced with `Symbol.for()`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for